setup.cfg

```diff
@@ -75,7 +75,7 @@ zip_safe = False  # https://mypy.readthedocs.io/en/latest/installed_packages.htm
 include_package_data = True
 python_requires = >=3.8
 install_requires =
-    numpy >= 1.19
+    numpy >= 1.19  # recommended to use >= 1.22 for full quantile method support
```

Comment:
I am slightly wary of this - I guess there is a lot of details that are not working on older supported versions. But ok to leave it.

Reply:
yeah me too. this is just another thing to keep updating for what is a standard assumption in the python ecosystem.

Reply:
Yeah. It was just specifically requested in the issue.
Feel free to propose a more general comment.

Reply:
How about
```
# minimum supported versions - install the newest version for best support
```
?

Reply:
Let's ping the person that created the issue https://github.com/smoors.
I never look into the setup file when installing packages, so better to ask someone who does?

Reply:
thanks a lot for doing this!

I actually prefer the original comment with the specific minimum version and reason why. without that info, it's basically useless.

also, don't think it's a maintenance burden at all. once the numpy requirement is bumped to >= 1.22 or higher, you can simply remove the comment.

but I'm not an `xarray` developer, so ... :)

Reply:
It's not this comment per se, but the expectation to list the compatibilities in setup.py. But anyhow - I suggest to merge as is with the disclaimer that this is not something we generally plan to do. Objections?